java/src/client/game/GameManager.java

```diff
@@ -54,6 +54,7 @@
      * @post This provides valid operations on GameManager
      */
     GameManager() {
+        clientModel = new ClientModel();
```

Comment:
If our stuff is breaking because it's null that means we're accessing it when we shouldn't. Usually in tests when the setup is not correct.